organizations/plugins.go

```diff
@@ -61,6 +61,17 @@ func (oh *OrganizationHandler) AddOrganizationPlugin(w http.ResponseWriter, r *h
 		return
 	}
 
+	var member Member
+	if err = utils.ConvertStructure(user, &member); err != nil {
+		utils.GetError(errors.New(err.Error()), http.StatusInternalServerError, w)
+		return
+	}
+
+	if member.Role != AdminRole {
+		utils.GetError(errors.New("member must be an admin"), http.StatusBadRequest, w)
```

Comment:
Let's have "Access denied" would here instead of "member must be an admin" then change the http status too to 403.